src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1507,14 +1507,14 @@ internal bool CanToggleFullscreenWatchShowing(object parameter)
 
         public void ToggleBackgroundGridVisibility(object parameter)
         {
-            if (!CanToggleBackgroundGridVisibility(parameter)) return;
+            if (BackgroundPreviewViewModel == null || !BackgroundPreviewViewModel.Active) return;
 
             BackgroundPreviewViewModel.IsGridVisible = !BackgroundPreviewViewModel.IsGridVisible;
         }
 
         internal bool CanToggleBackgroundGridVisibility(object parameter)
         {
-            return BackgroundPreviewViewModel != null && BackgroundPreviewViewModel.Active;
```

Comment:
`IsEnabled` of corresponding menu item is already bound to `BackgroundPreviewViewModel.Active`. So this is redundant